src/e2e-app/virtual-scroll/virtual-scroll-e2e.html

```diff
@@ -1,6 +1,6 @@
 <section class="demo-virtual-scroll-uniform-size">
   <h3>Uniform size</h3>
-  <cdk-virtual-scroll-viewport class="demo-viewport" autosize>
+  <cdk-virtual-scroll-viewport class="demo-viewport" autosize tabindex="0">
```

Comment:
Why the `tabindex` here?

Reply:
Axe complains that scrollable areas need to have tab stops.